### DISCUSSION TITLE
Error on Deploy

### DISCUSSION THREAD

**Discussion by martinslae** [community/community#51993] [Actions] [answered]
### Select Topic Area

Bug

### Body

I am getting the message when I go to deploy my application, even I re-run some already deployed application

**GitHub Actions has encountered an internal error when running your job.**
and some times
**An error occurred while provisioning resources (Error Type: Failure).**

The error raises after about 5 seconds from starting to deploy

There's some issue with Actions?

**Comment by Akash1134** [comment]
Hi @martinslae ,

Are you facing this issue across multiple repos?  Specific instances of workflow runs would be great as well (if possible,  just provide the link to the workflow run)

**Reply by martinslae** [reply]
Hi @Akash1134 
So, after dozens of tries is not raising error but because the deploy not starts never. It's  waiting for 15 hours, the message is **"Waiting for a runner to pick up this job..."**  -  I had cancel and start again many times but now I am waiting, for 15 hours.....


This is my workflows file:

`
name: API - Development

on:
  push:
    branches: [Dev]
    paths:
      - "src/**"
      - ".github/**"
      - "app.yaml"
    
  workflow_dispatch:
env:
  DAY_OF_WEEK: ${{ github.ref_name }}
  reason: ${{ github.event_name }}
  job_name: ${{ github.repository }}
  branch: ${{ github.action }}
  iclass: "B8"

jobs:
  build-deploy:
    runs-on: ubuntu-18.04

    steps:
      - uses: actions/checkout@v2

      - name: Install & Build
        run: |
          echo ${{env.DAY_OF_WEEK}}
          echo um.  ${{env.job_name}}
          echo dois. ${{env.reason}}
          echo tres. ${{env.branch}}
          npm install
          npm run build
          npm i --save replace-in-file
          node ./app.yaml.make B8
        
      - id: Deploy
        uses: google-github-actions/deploy-appengine@v0
        with:
          credentials: ${{ secrets.GCP_CORES_APP_DEV_SA }}
`

**Comment by martinslae** [comment, marked as answer]
Hey guys, 
I figure out the issue: The action does not catch the deploy because the `runs-on:`, was `ubuntu-18.04`.
I must to change to `ubuntu-latest` but I got an error on deploy because of the node version(12), then I changed to node16, and with both configurations, it works, finally!

The weird is we should get some information about and not be blind about the issue.

Thanks
